lib/cfa/augeas_parser.rb

```diff
@@ -84,6 +84,16 @@ def initialize(tree, value)
     def [](key)
       tree[key]
     end
+
+    def ==(other)
+      other.class == self.class &&
+        other.value == value &&
+        other.tree == tree
```

Comment:
this one can look like `[:class, :value, :tree].all? { |a| self.public_send(a) == other.public_send(a)}` which makes it easier to extend it if more attributes need to be compared.